K2Bridge/Visitors/QueryStringClauseVisitor.cs

```diff
@@ -67,7 +67,7 @@ public void Visit(QueryStringClause queryStringClause)
                     break;
 
                 case QueryStringClause.Subtype.Phrase:
-                    queryStringClause.KustoQL = $"{queryStringClause.ParsedFieldName} {KustoQLOperators.Contains} \"{queryStringClause.Phrase.EscapeSlashes()}\"";
+                    queryStringClause.KustoQL = $"{queryStringClause.ParsedFieldName} {KustoQLOperators.Contains} \"{queryStringClause.Phrase}\"";
```

Comment:
Isn't it required to escape here (why was it removed)?

Reply:
This method works when you enter your filter in "query" bar:
![image](https://user-images.githubusercontent.com/464801/93665364-8ad10980-fa7e-11ea-8f74-3cac7f6d6aa4.png)
So, if we want to search for `Coats with "quotes"`, we have to escape doublequote **inside** Kibana query bar, because doublequotes are used by Kibana to mark the boundaries of phrases (like -field:"abc").
This `EscapeSlashes` method escapes the kibana doublequote escape and the query fails:
![image](https://user-images.githubusercontent.com/464801/93665400-d2f02c00-fa7e-11ea-9dec-718a5b607556.png)

Without these methods those slashes escaping the doublequotes are handed to Kusto and it works.

While writing this I understood that we also have to remove EscapeSlashes at line 64 so `QueryStringClause.Subtype.Term` wouldn't be also double escaped. And on line 85 so `hasprefix` would also work with doublequotes:
![image](https://user-images.githubusercontent.com/464801/93665571-244ceb00-fa80-11ea-8461-d5fb70d24a1c.png)

Maybe I miss some other case where we actually need escaping slashes inside kibana query bar?

Reply:
Thank you @spacentropy for these details. 
I understand what you aim to accomplish here but my concern is that your change reverses something else, specifically how to handle _phrases_ that include a **\** char.  taking your example, this phrase will now fail: "Coats with \slash\" (or not return the right result).
I'm sure there's a way to fix that too, but before we go that route - keeping K2 behavior on-par with elastic is/was really hard. Can you share a comparable example from Kibana working with Elaticsearch directly?